File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1113.cs
using System;

class URI
{
    static void Main() {
        
        int x, y;
        while(true){
            string n = System.Console.ReadLine().Trim();
            string[] ns = n.Split(' ');
            x = int.Parse(ns[0]);
            y = int.Parse(ns[1]);
            if(x==y){
                break;
            }
            if(x>y){
                Console.Write("Decrescente\n");
            } else{
                Console.Write("Crescente\n");
            }
        }
    
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1113.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

while(true){
    ns = lines.shift().split(' ')

    x = parseInt(ns[0])
    y = parseInt(ns[1])
    if(x==y){
        break;
    }
    if(x>y){
        console.log("Decrescente")
    } else{
        console.log("Crescente")
    }
}